homeassistant/components/iotawatt/sensor.py

```diff
@@ -27,17 +27,11 @@
 from homeassistant.helpers import entity, entity_registry
 from homeassistant.helpers.device_registry import CONNECTION_NETWORK_MAC
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
-from homeassistant.helpers.restore_state import RestoreEntity
 from homeassistant.helpers.typing import StateType
 from homeassistant.helpers.update_coordinator import CoordinatorEntity
 from homeassistant.util import dt
 
-from .const import (
-    ATTR_LAST_UPDATE,
```

Comment:
Seems that `ATTR_LAST_UPDATE` isn't used anywhere anymore at this point, so it should be removed from the const file.